src/services/repository/app/repository.ts

```diff
@@ -37,4 +45,43 @@ export class Repository implements ForManagingUser {
       email: newUser.email,
     };
   }
+
+  async getInternalUser(email: string): Promise<RepoUser> {
+    const user = this.userList.find((user) => user.email === email);
```

Comment:
Agregar separación entre unidades lógicas
```suggestion
    const user = this.userList.find((user) => user.email === email);
    
```

Reply:
no sabía muy como interpretar lo de la separación entre unidades lógicas, asumí que te referías a que en el repository estaba combinando consultas de permissions y user, puede que sea por ahí?

Reply:
Es solo el agregado de un espacio entre unidades. Aquí tienes un video como guía: 

👨‍💻💻 Programando con estilo: cómo escribir código limpio y elegante como un verdadero 😎crack💪
https://youtu.be/041fWH35X-4